translations/en.json

```diff
@@ -110,6 +110,12 @@
   "map.select.layer": "Select the layer to display",
   "nav.back": "Back",
   "next": "next",
+  "organisation.sort.intro": "The organisations are...",
+  "organisation.sort.sortBy": "Sort by:",
+  "organisations.sortBy.nameAsc": "Name A > Z",
+  "organisations.sortBy.nameDesc": "Name Z > A",
+  "organisations.sortBy.recordCountAsc": "Publications (ASC)",
+  "organisations.sortBy.recordCountDesc": "Publications (DESC)",
```

Comment:
Here I'd rather use "Name (ascending)", "Name (descending)", "Records (ascending)" and "Records (descending)", what do you think?

Reply:
Hum, I followed `Name A > Z` from the mockups (which also seems a little more intuitive to me), but you are right that `ascending/descending` for all four is more consistent and more intuitive that `ASC/DESC`, I'll adapt that

Reply:
Hum, when testing I'm not a big fan of the repetitive `ascending/descending`, actually. I propose just to replace the `ASC/DESC` by `ascending/descending`.

Reply:
makes sense, why not "A → Z" for names and "0 → 9" for records? or "A ➡ Z"? (unicode is useful for symbols which are not icons)

Reply:
ah, good idea, I'll go for that, thanks!